syne_tune/optimizer/baselines.py

```diff
@@ -0,0 +1,96 @@
+from typing import Dict
+
+from syne_tune.optimizer.schedulers.fifo import FIFOScheduler
+from syne_tune.optimizer.schedulers.hyperband import HyperbandScheduler
+from syne_tune.optimizer.schedulers.multiobjective.moasha import MOASHA
+from syne_tune.optimizer.schedulers.pbt import PopulationBasedTraining
+
+
+class RandomSearch(FIFOScheduler):
+    def __init__(self, config_space: Dict, metric: str, **kwargs):
+        super(RandomSearch, self).__init__(
+            config_space=config_space,
+            metric=metric,
+            searcher="random",
+            **kwargs,
+        )
+
+
+class BayesianOptimization(FIFOScheduler):
+    def __init__(self, config_space: Dict, metric: str, **kwargs):
+        super(BayesianOptimization, self).__init__(
+            config_space=config_space,
+            metric=metric,
+            searcher="bayesopt",
+            **kwargs,
+        )
+
+
+class ASHA(HyperbandScheduler):
```

Comment:
I'd also include type ('stopping' or 'promotion') in the list of named args. Of course, I can get it with **kwargs, but it is certainly important to have that selection.

An alternative is to have ASHAStopping, ASHAPromotion,

or AsyncSHStopping, AsyncSHPromotion

Reply:
One thing about max_t.

This is actually not a mandatory parameter. If you set max_resource_attr, this value is obtained from config_space, which is often safer than to pass it again.

Reply:
Oh, and finally, I am not sure we should not include grace_period (with default 1). After all, this is quite an important parameter.

It can also remain in **kwargs.

Maybe it all depends how good the docs are for these baselines. We should tell people about which parameters are potentially important.

Reply:
The rationale is to include only required parameters here. While `max_t` is not always required, it is required sometimes so it also went into this category.

Reply:
I propose as a first step to have only non-default parameters. We can certainly revise that if we want to put up more parameters (I guess the same question may apply to kwargs when using our schedulers).